dnsapi/dns_googledomains.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env sh
+
+# Author: Alex Leigh <leigh at alexleigh dot me>
+# Created: 2023-03-02
+
+#GOOGLEDOMAINS_ACCESS_TOKEN="xxxx"
+#GOOGLEDOMAINS_ZONE="xxxx"
```

Comment:
Where does one find their managed google domains 'zone'?

Reply:
There is no need for specifying a domain zone, zone is the domain for which you need the certificate.